Database Scripts/V006__create_table_enemy_action.sql

```diff
@@ -1,13 +1,8 @@
-USE BeantasticDB;
-GO
-
-CREATE TABLE [dbo].[EnemyAction](
-	[enemyActionId] [INT] IDENTITY(1,1) NOT NULL,
-	[enemyId] [INT] NOT NULL,
-    [ActionTypeId] [INT] NOT NULL,
-	[name] [VARCHAR] (30) NOT NULL,
-	[description] [VARCHAR] (100) NOT NULL
-	PRIMARY KEY CLUSTERED (enemyActionId),
-	FOREIGN KEY (actionTypeId) REFERENCES [dbo].[ActionType](actionTypeId)
+CREATE TABLE EnemyAction (
+    enemyActionId SERIAL PRIMARY KEY,
+    enemyId INT NOT NULL,
+    ActionTypeId INT NOT NULL,
```

Comment:
Think only thing is to make this _actionTypeId_ and then also remove the capital A when you add the foreign key a couple lines below.

Reply:
Ah, chatgpt, you've let me down. I fixed it. Thanks

Reply:
@Dutchua didn't the enemyAction change to action? Or did we change it back to enemyAction?